### ISSUE TITLE
An assembly specified in the application dependencies manifest (SIL.NuGetCleaner.deps.json) was not found

### ISSUE THREAD

**Issue by thomasgalliker** [sillsdev/SIL.NuGetCleaner#46]
Hi! When I run nugetclean, it complains that there is some dependency missing. What could be the reason? Thanks for your help
```
C:\>nugetclean --api-key=aaaa --dry-run PushNotifications.AspNetCore
Error:
  An assembly specified in the application dependencies manifest (SIL.NuGetCleaner.deps.json) was not found:
    package: 'System.Security.AccessControl', version: '4.7.0'
    path: 'runtimes/win/lib/netcoreapp2.0/System.Security.AccessControl.dll'

```

**Comment by ermshiperete** [comment]
Thanks for reporting! 

This has been fixed in version 1.3.1.

**Comment by thomasgalliker** [comment]
Thanks alot!
